e2e-win/mise_hook.Tests.ps1

```
@@ -0,0 +1,9 @@
+Describe 'mise_hook' {
+    It 'doesn''t clobber $LASTEXITCODE' {
+        mise activate pwsh | Out-String | Invoke-Expression
```

PSScriptAnalyzer (1.25.0)

[warning] Invoke-Expression is used. Please remove Invoke-Expression from script and find other options instead.

(PSAvoidUsingInvokeExpression)